nossomercadolivre-api/src/main/java/com/github/nossomercadolivre/validation/UniqueNameCategoryException.java

```diff
@@ -0,0 +1,8 @@
+package com.github.nossomercadolivre.validation;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(code = HttpStatus.BAD_REQUEST)
+public class UniqueNameCategoryException extends RuntimeException {
```

Comment:
Onde usamos? não to lembrado.

Reply:
ahhhhhh estava experimentando isso subiu isso sem precisar.